nbinteract/plotting.py

```diff
@@ -75,6 +76,11 @@
         'Normalize histogram area to 1 if True. If False, plot '
         'unmodified counts. (default True)'
     ),
+    'marker':(
+        'Shape of marker plots. Possible values: {‘circle’, ‘cross’, '
+         '‘diamond’, ‘square’, ‘triangle-down’, ‘triangle-up’, ‘arrow’, '
+         '‘rectangle’, ‘ellipse’}'
```

Comment:
Could we change these smart quotes to double quotes? Otherwise they'll break when people copy-paste them.

Reply:
Changes made!